openqaoa/optimizers/optimization_methods.py

```diff
@@ -20,6 +20,8 @@
 import numpy as np
 from scipy.optimize import OptimizeResult
 
+import time
```

Comment:
I think this import isnt needed for this file

Reply:
erased